include/picongpu/fields/currentDeposition/EmZ/EmZ.hpp

```diff
@@ -101,24 +102,24 @@ namespace picongpu
                 /* calculate positions for the second virtual particle */
                 for(uint32_t d = 0; d < simDim; ++d)
                 {
-                    line.m_pos0[d] = calc_InCellPos(posStart[d], I[0][d]);
-                    line.m_pos1[d] = calc_InCellPos(relayPoint[d], I[0][d]);
+                    line.m_pos0[d] = posStart[d] - shiftStart[d];
```

Comment:
The for loop should not contain a doc string that the in cell position is calculated. Before it was obvious because of the function name.

Reply:
I can add it. However I think if anything now it's more clear than before due to changed naming

Reply:
Yes please add a doc string or reintroduce a helper function.

Reply:
Added a comment (extended one just before the loop)